### ISSUE TITLE
Why the developer is not answering the issues when exe's are flagged by anti virus. Can't that issue be fixed?

### ISSUE THREAD

**Issue by pushpeshkarki2016** [r00tmebaby/LinkedIn-Downloader#32]
Why the developer is not answering the issues when exe's are flagged by anti virus. Can't that issue be fixed by proper signing the .exe instead the developer is asking to not use the software. i hope we as users are asking to look into it. if you can't provide solution accept it and we will not use your software.

**Comment by djoesch** [comment]
Because not any single freeware that does remote routines is known by Anti Virus software.
Just because somebody says you look ill, you won't sue the doctor... will you?

**Comment by fastmarketo** [comment]
> Why the developer is not answering the issues when exe's are flagged by anti virus. Can't that issue be fixed by proper signing the .exe instead the developer is asking to not use the software. i hope we as users are asking to look into it. if you can't provide solution then accept it and i will not use your software.

Don't get angry. Understand his contribution is voluntarily and he may have other commitments. Just be polite and request if he can reply and fix the issue. Understand life and be practical. We are not paying him single dime for his contribution.

**Comment by r00tmebaby** [comment]
Hello,

Thanks for your polite words @fastmarketo, and @djoesch trying to defend me. I need no defence and have actually answered and closed that question a while ago. Another person has mentioned that there is a software that may steal your Linkedin credentials under my youtube video.

Well, everyone is free to believe whatever he/she wants. This software (it's not really a big software but let say a small tool) have already code-shared, so anyone who does understand codding can simply analyze the code and see if there is such harmful software built-in.

In facts, the reason why some AV can give you false-positive report is that the last executable is packed with virtualization software which block any attempt to debug the program and since the AV try to analyze it but can not simple report that it might be a harmful software. You are free to use or not use the tool and if you feel unsafe, just don't use it. So no matter what I say here I can not persuade you in the opposite, right?

In any case before the last version .16 all other works and code has been shared, so you can use the python version instead of the executable one (of course without some of the new features introduced in the latest version ).

At a certain point, I may release the latest source code but some if you may find difficulties to use console and python apps. That was the reason to create an executable and spent time with building UI that can be simply executed than a console app, to make it easier for you. Well, if you do not appreciate my work or feel unsafe as explained above, just don't use the software.

As a developer and security specialist, I can tell you that sharing code on internet especially on public places like Github, Youtube etc that can harm someones PC or worst to steal a piece of private information is an offence act and trust me that no one who has a picture, professional (which means checked) profile will risk to get banned or prosecuted for such actions. There are so many much easier, faster, proven and stable methods that can be used to get someones private information than spending time to write a proper program that helps people.

I expect that you will no longer post requests in this matter that are damaging my authority because for me the easiest way is to just remove the program and all that think is helpfull will struggle because of people like you.

**Comment by fastmarketo** [comment]
@r00tmebaby I hope you will be able to get out of one cynical comment and keep up the good work. we send you +ve Karma. Thanks.

**Comment by djoesch** [comment]
I really appreciate the work. At the moment I think LinkedIn has changed the names of some CSS classes, so it does not work any more. But I liked the way, your App worked and it was a lot of fun to try it. Thank you.

**Comment by fastmarketo** [comment]
> I really appreciate the work. At the moment I think LinkedIn has changed the names of some CSS classes, so it does not work any more. But I liked the way, your little App was designed and it was a lot of fun to try it. Thank you.

Yes, and they are also manipulating layouts. But the software was excellent to work with. Just a little patience needed.

**Comment by r00tmebaby** [comment]
I will make it work again and add the features that were asked.
